go.mod

```diff
@@ -12,12 +12,14 @@ require (
 	github.com/chainguard-dev/terraform-infra-common v0.6.85
 	github.com/cloudevents/sdk-go/v2 v2.15.2
 	github.com/coreos/go-oidc/v3 v3.11.0
+	github.com/go-jose/go-jose/v3 v3.0.3
```

Comment:
can we use ` "github.com/go-jose/go-jose/v4" `

Reply:
Done!